pkg/ipamd/ipamd.go

```diff
@@ -2001,6 +2001,11 @@ func (c *IPAMContext) AnnotatePod(podName string, podNamespace string, key strin
 			if err == nil && pod == nil {
 				log.Warnf("get a nil pod for pod name %s and namespace %s", podName, podNamespace)
 			}
+			// since the GetPod() error has been decorated, we have to check key words
```

Comment:
I would improve the comment to something like this to be more clear. But we can do as a follow up.

`releasedIP` is only set in the DelNetwork RPC call. We don’t want to error out on removing PodAnnotation when Pod is deleted and IP is being released.